### PR TITLE
Skip cleanup test in downstream run

### PR DESCRIPTION
**Pull request by akus062381**

### CODE REVIEW

**Review by nixocio — DISMISSED** [review]

**Review by nixocio — DISMISSED** [review]

File: cypress/e2e/awx/cleanup/awx-cleanup.cy.ts [hunk 1]
```diff
@@ -32,16 +33,6 @@ describe('AWX Cleanup', () => {
...
-    cy.requestGet<AwxItemsResponse<Organization>>(
-      awxAPI`/organizations?name__startswith=E2E&page=1&page_size=200&created__lt=${tenMinutesAgo}`
-    ).then((result) => {
-      for (const resource of result.results ?? []) {
-        cy.deleteAwxOrganization(resource, { failOnStatusCode: false });
-      }
-    });
-  });
-
   it('cleanup users', () => {
```
Comment (vidyanambiar) [inline cypress/e2e/awx/cleanup/awx-cleanup.cy.ts:36]: Can we also skip cleaning up AWX users please?

File: cypress/e2e/hub/overview/hub-overview.cy.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const collectionNames: { [key: string]: { [key: string]: string } } = {
   },
 };
 
-describe.skip('HUB Overview', () => {
+describe('HUB Overview', () => {
```
Comment (nixocio) [inline cypress/e2e/hub/overview/hub-overview.cy.ts:17]: Are we ready to be re-enable this test?

**Review by nixocio — DISMISSED** [review]
@akus062381 other than the HUB test being re-enable question. 👍

**Review by vidyanambiar — DISMISSED** [review]

**Review by nixocio — DISMISSED** [review]

**Comment by akus062381** [comment]
closing as obsolete

**Pull request closed without merging**